pye3d/detector_3d.py

```diff
@@ -544,6 +544,8 @@ def _prepare_result(
             projected_pupil_circle = Ellipse(np.asarray([0.0, 0.0]), 0.0, 0.0, 0.0)
 
         result["ellipse"] = ellipse2dict(projected_pupil_circle)
+        result["location"] = result["ellipse"]["center"]  # pupil center in pixels
+
         result["diameter"] = projected_pupil_circle.major_radius
```

Comment:
Same as above. Also, is the major_radius really the diameter? Or half of the diamter?

Reply:
@euryalus The naming suggests it is half the diameter. Is there an easy way to check?

Reply:
Have looked in the code. Here, the major_radius is the diameter.

Reply:
```suggestion
        # projected_pupil_circle is an OpenCV ellipse, i.e. major_radius is major diameter
        result["diameter"] = projected_pupil_circle.major_radius
```